kernel/declarations.mli

```diff
@@ -164,6 +164,15 @@ type regular_inductive_arity = {
 
 type inductive_arity = (regular_inductive_arity, template_arity) declaration_arity
 
+type squash_info =
+  | AlwaysSquashed
+  | SometimesSquashed of Sorts.Quality.t list
```

Comment:
Why not a set here? It makes it clear that we care neither about order nor duplication.

Reply:
changed